gtsfm/frontend/inlier_support_processor.py

```diff
@@ -75,7 +78,8 @@ def run(
             i2Ri1 = None
             i2Ui1 = None
             v_corr_idxs = np.array([], dtype=np.uint64)
-            return i2Ri1, i2Ui1, v_corr_idxs, None
+
+            return i2Ri1, i2Ui1, v_corr_idxs, TwoViewEstimationReport(v_corr_idxs=v_corr_idxs, num_inliers_est_model=0)
```

Comment:
I do not get your point. Can you elaborate.

I saw these errors: 
```
File "/home/runner/work/gtsfm/gtsfm/gtsfm/averaging/rotation/cycle_consistency.py", line 312, in _compute_metrics
    if report.R_error_deg is None or report.U_error_deg is None:
AttributeError: 'NoneType' object has no attribute 'R_error_deg'
```
but maybe that is because i was returning a report with null values instead of returning None itself.

Reply:
So we check for None here: 
https://github.com/borglab/gtsfm/blob/master/gtsfm/two_view_estimator.py#L256

So we need some signal that we are discarding. Otherwise the statistics will get polluted with zeros for pairs we are not actually using.

Reply:
Why are we setting `num_inliers_est_model` to 0 instead of `num_inliers_est_model`? Even though it doesn't have _enough_ outliers with respect to our threshold, it's kind of misleading to say it didn't find any.

Reply:
Ok I will revert to None.